src/ftest.jl

```diff
@@ -0,0 +1,130 @@
+type FTestTable
+    ntests::Int
+    SSR1::Array{Float64, 1} # sum of squared residuals, model 1
```

Comment:
This is just a stylistic thing, but typically we try to reserve caps for types and use all lowercase for variables and fields. So this should be `ssr1`, likewise for the other fields here.